Fix Addresses list

pmknutsen opened the pull request:
Displaying all addresses may be slow and prone to hang the UI when there is a very large number of wallet addresses. Furthermore, users should be discouraged to re-use addresses. This PR improves the listing of addresses:
- Hide wallet addresses by default
- Display only new address when generated
- Add button to display all addresses
- Add copy-to-clipboard icon next to addresses

lukechampine commented:
hmm. Did you see my comments on #89? siad should be executed with ./Sia as its working directory. I believe this means you want to spawn the process like so: `child_process.spawn("./siad", {cwd: "./Sia")})` (See actual implementation [here](https://github.com/NebulousLabs/Sia-UI/blob/master/lib/daemon.js#L35).)

zlalvani commented:
Alright, we can ignore the request then.

pmknutsen committed:
Hide wallet addresses by default
Add copy-to-clipboard icon next to addresses
Add button to display all addresses

DavidVorick commented on app/plugins/Wallet/css/wallet.css:
```diff
@@ -218,9 +218,15 @@
 .entry .address {
 	display: inline-block;
-	width: 80%;
+	width: calc(80% - 25px);
```
not a good js dev, but is this potentially going to cause problems on smaller or larger screens?

pmknutsen replied:
I don't see how. The icon column is set to 25% pix and this `calc` just computes the correct width of the addresses column.
